include/mbedtls/error.h

```diff
@@ -52,7 +52,7 @@
  * GCM       2  0x0012-0x0014
  * BLOWFISH  2  0x0016-0x0018
  * THREADING 3  0x001A-0x001E
- * AES       2  0x0020-0x0022
+ * AES       2  0x0020-0x0022   0x006E-0x006E
```

Comment:
It is supposed to be a range of values that effectively only contains 1 value, namely 0x006E. I simply followed the same convention that is used throughout this file. For example, check [this line](https://github.com/ARMmbed/mbedtls/pull/964/files#diff-c52ac35237c20a39a4621bdcaf1501ceR59).